vyxal/elements.py

```diff
@@ -4263,8 +4277,7 @@ def f():
                     exhausted += 1
                 if exhausted == 2:
                     break
-                else:
-                    yield [left_item, right_item]
+                yield [left_item, right_item]
```

Comment:
This is definitely clearer as is.

```suggestion
                else:
                    yield [left_item, right_item]
```

BTW, I'm pretty sure I've previously rejected a change like this; possibly even this exact bit of code. Deepsource is just too aggressive with these complaints for my liking.

Reply:
I always feel return and yield without else make python code pythonicaler.

Reply:
Alright, I'll revert it. We should also add a comment to make Deepsource ignore it in the future.